Makefile.am

```diff
@@ -157,10 +157,6 @@ tests_multichan_throughput_test_SOURCES = tests/multichan_throughput_test.c test
 tests_multichan_throughput_test_CFLAGS = $(FFTW3_CFLAGS)
 tests_multichan_throughput_test_LDADD = src/libsamplerate.la $(FFTW3_LIBS)
 
-if HAVE_LIBSNDFILE
-check_PROGRAMS += tests/src-evaluate
```

Comment:
Err, no.

Reply:
Why? **ALL** your tests/examples are written in a way, that they compile to stubs if the required libs are not found. So why have a special case for this one? It works this way (proven by CI)

The current approach fails because of https://github.com/erikd/libsamplerate/pull/69/files/8f19f25358ead6fd7bb43f7099e228265b72ae3e..1c7b71e07e5610ea0be9504c485f687e9aec03d4#diff-c949f93d03f44a4217d7a138f9e2e54aR82

The program is unconditionally required and if libsndfile isn't found it breaks the built as no sources are added to that binary. The alternative to this change is to remove it from the linked `check_PROGRAMS` but after seeing the `#if HAVE_LIBSNDFILE` in the source, the indented way looks rather like I have done it here.

Reply:
Pulled into separate PR #73 to fix bug described here